docs/themes/oso-webpack/index.js

```diff
@@ -356,3 +356,23 @@ function makePromptsUnselectable() {
     });
   })
 };
+
+window.recordFeedback = (isUp) => {
+  recordFeedback(isUp).then(() => {
```

Comment:
Could we disable the input before the request is complete? The notion recorder can be a little slow so this would give more immediate feedback to the user. Ideally we'd have a submitting state but I don't think that's really necessary right now.